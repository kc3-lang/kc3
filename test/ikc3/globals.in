quote __DIR__
__DIR__
quote __FILE__
__FILE__
