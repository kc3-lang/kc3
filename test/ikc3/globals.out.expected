__DIR__
"./"
__FILE__
"globals.kc3"
